src/jest-types.ts

```diff
@@ -60,7 +60,7 @@ export interface FullJestConfig {
   roots: Path[];
   runner: string;
   setupFiles: Path[];
-  setupTestFrameworkScriptFile: Path;
+  setupTestFrameworkScriptFile: Path | null;
```

Comment:
This should originally have been `setupTestFrameworkScriptFile?: Path;`

Reply:
haha indeed it looks a bit confused to me when checking their types.

Reply:
oh right .. I guess I saw the other one. I think this should be fine :)